gandalf.js

```diff
@@ -1,13 +1,14 @@
 /**
```

Comment:
SCRIPT-This new feature lacks unit tests. Could you add some to ensure everything works correctly?